topics.py

```diff
@@ -37,6 +37,7 @@ class Camera(Topic):
     get_overlays = 'camera/get_overlays'
     set_overlay = 'camera/set_overlay'
     push_overlays = 'camera/push_overlays'
+    errors = 'camera/errors'
```

Comment:
I reckon keep the `/v3/` prefix. We should fix the rest of those topics to also have the prefix in a future PR